src/guide/reactivity-computed-watchers.md

```diff
@@ -211,6 +213,37 @@ firstName.value = 'John' // logs: ["John", ""] ["", ""]
 lastName.value = 'Smith' // logs: ["John", "Smith"] ["John", ""]
 ```
 
+However, if you are changing both watched sources simultaneously in the same method, the watcher will be executed only once:
+
+```js{9-13}
+setup() {
+  const firstName = ref("");
+  const lastName = ref("");
+
+  watch([firstName, lastName], (newValues, prevValues) => {
+    console.log(newValues, prevValues);
+  });
+
+  const changeValues = () => {
+    firstName.value = "John";
+    lastName.value = "Smith";
+    // logs: ["John", "Smith"] ["", ""]
+  };
+
+  return { changeValues };
+}
+```
+
+If you want watcher to be triggered after every change, make sure to add the [nextTick](/api/global-api.html#nexttick) call between changes:
```

Comment:
```suggestion
Note that multiple synchronous changes will only trigger the watcher once.

If you want the watcher to be triggered after every change, make it synchronous (`sync: true`, not recommmended), or make sure to wait a tick  between each change, i.e. by using [nextTick](/api/global-api.html#nexttick):
```

This might cover a bit more ground. not 100% happy with the wording, maybe someone else can refine?

Reply:
@LinusBorg added the suggestion and unresolved the conversation so others can chime in! 👍🏻